recipes/dmgbuild/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "dmgbuild" %}
+{% set version = "1.6.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 7ced2603d684e29c22b4cd507d1e15a1907e91b86259924b8cfe480d80553b43
+
+build:
+  number: 0
+  skip: true  # [not osx]
```

Comment:
If this is a macOS only tool we should make it a OS-noarch package. See https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-packages-with-os-specific-dependencies

Reply:
You will need to add the conda-forge.yml file. Please read the link above.

Reply:
Please check if it is correct.